src/commodore/index.ts

```diff
@@ -280,10 +282,12 @@ export async function extractPackageFile(
     return null;
   }
 
+  const githubBaseUrl = extraConfig.githubBaseUrl;
   const deps = components.map((v: CommodoreDependency) => ({
     depName: `${v.name} in ${fileName}`,
-    packageName: v.url,
+    packageName: v.url.startsWith(githubBaseUrl) ? v.url.slice(githubBaseUrl.length).replace(/\.git$/, '') : v.url,
     currentValue: v.version,
+    datasource: v.url.startsWith(githubBaseUrl) ? githubRelease.GithubReleasesDatasource.id : gitRef.GitRefsDatasource.id,
```

Comment:
What does the GitHub releases datasource return for GitHub repos which don't have releases, but have tags?

Note that this is unlikely to be a problem since the component template auto-generates releases from tags, but it would be nice to note any new restrictions on the functionality of the commodore manager in `src/commodore/readme.md`